modules/openy_features/openy_location/modules/openy_loc_facility/config/install/core.entity_view_display.node.facility.default.yml

```diff
@@ -28,6 +28,14 @@ targetEntityType: node
 bundle: facility
 mode: default
 content:
+  addthis:
```

Comment:
I don't like dependency from addthis here
Could we remove it?
add this should be on blogs, news, probably Landing Pages. But Facility  - no need